docs/src/APIs/Soil.md

```diff
@@ -57,6 +57,8 @@ ClimaLSM.Soil.FreeDrainage
 ClimaLSM.Soil.AtmosDrivenFluxBC
 ClimaLSM.Soil.AbstractSoilSource
 ClimaLSM.Soil.PhaseChange
+ClimaLSM.Soil.vanGenuchten
```

Comment:
I think `AbstractSoilHydrologyClosure` should be added here too

Reply:
added it!